shadowsocks-csharp/Model/Configuration.cs

```diff
@@ -78,6 +82,10 @@ public static Configuration Load()
                 string configContent = File.ReadAllText(CONFIG_FILE);
                 Configuration config = JsonConvert.DeserializeObject<Configuration>(configContent);
                 config.isDefault = false;
+                if (UpdateChecker.Asset.CompareVersion(UpdateChecker.Version, config.version) > 0)

```

Comment:
If user is updated from a legacy version which does not contains 'version' element in `gui-config` file. The code should still work.